tide/master/rest/RestServer.cpp

```diff
@@ -53,7 +60,7 @@ RestServer::RestServer(const int port)
     : zeroeq::http::Server{zeroeq::URI{QString(":%1").arg(port).toStdString()}}
 {
     _init();
-    addToWhitelist("127.0.0.1");
+    _whitelist.insert("127.0.0.1");
```

Comment:
this should must be in init(), otherwise it is not called by the default constructor!

Or more simply do it in the header:
`std::set<std::string> _whitelist{{"127.0.0.1"}};`

Or even simpler, since there is just one entry no need for a set:
```
bool RestServer::_isWhitelisted(const std::string& source) const
{
    return _getHostname(source) == "127.0.0.1";
}
```